README.md

```diff
@@ -12,15 +12,42 @@ work!
 - [Drush](https://github.com/drush-ops/drush-launcher#installation---phar)
 - SQLite and the related PHP extensions
 
-## Usage
+## Initializing a new project
 
-To create a new Tome project, run:
+To create a new Tome project in the `my_site` subdirectory, run:
+
+```bash
+mkdir my_site
+cd my_site
+composer create-project drupal-composer/drupal-project:8.x-dev . # <-- the dot at the end means current directory
+composer require drupal-tome/tome-project
+composer update
+```
+
+Create new SQL database `testdb`, with its own separate user `tome` that has all privileges for that database and only for that database, with a password `mypassword`:
```

Comment:
This project is set up to use SQLite by default, which should work. If you want to add alternative instructions for MySQL I would put that under another heading as it should not be required.